toolsrc/src/vcpkg/base/strings.cpp

```diff
@@ -182,6 +182,15 @@ std::vector<std::string> Strings::split(StringView s, const char delimiter)
     }
 }
 
+std::vector<std::string> Strings::split_paths(StringView s)
+{
+#if defined(_WIN32)
+    return Strings::split(std::move(s), ';');
+#else // ^^^ defined(_WIN32) // !defined(_WIN32) vvv
+    return Strings::split(std::move(s), ':');
```

Comment:
```suggestion
    return Strings::split(s, ';');
#else // ^^^ defined(_WIN32) // !defined(_WIN32) vvv
    return Strings::split(s, ':');
```

`s` is a `StringView`, there's no point to `move`-ing it.